client/src/services/utilities.js

```diff
@@ -167,6 +167,14 @@ export function showRelationType(typeId: number, typeText: string) {
   return `${typeText || 'Neznáme'} ${typeId > 0 ? '>' : '<'}`
 }
 
+export function getRelationTitle(typeId: number, name1: string, name2: string) {
```

Comment:
I find it a bit strange that this function is in a common `utilities` file, when it's used only in one place and probably will be used only in one module ever... I know there are many others like it, but please, could you create a separate utils file in `Relation` module? This file is starting to be a bit clumsy. Thanks a lot :muscle: